pkg/utils/parse_arf_result.go

```diff
@@ -656,6 +663,14 @@ func nameFromId(scanName, ruleIdRef string) string {
 	return fmt.Sprintf("%s-%s", scanName, dnsFriendlyFixId)
 }
 
+// get rule name without the scan name prefix
+func ruleNameFromId(ruleIdRef string) string {
```

Comment:
We have the same code in `IDToDNSFriendlyName` (ccr method). Can we maybe converge them, perhaps using a function in utils?